docs/user/installation.rst

```diff
@@ -149,6 +149,29 @@ Navigate to your preferred installation directory, i. e. use ``mkdir ~/intelmq &
 
 Your installation should be successful now. You're now able to visit ``http://127.0.0.1:1337/`` to access the intelmq-manager.
 
+Docker without docker-compose
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Navigate to your preffered installation directory, i. e. use ``mkdir ~/intelmq && cd ~/intelmq``
+
```

Comment:
s/preffered/preferred and does docker really have to be started from a specific directory?

Reply:
We don't start docker, we just prepare a setup & directory to work with. I. e. mount configs and logs.